setup.py

```diff
@@ -89,11 +89,13 @@ def package_files(directory: str):
     'myst-parser>=0.15.2',
     'pylint>=2.12.2',
     'docformatter>=1.4',
+    'psutil>=5.8.0',
```

Comment:
The error messages mention to install composer with perf...could we defer the refactoring of the imports to another PR?
(I agree we have wayyyyy tooo many extra dep sections)

Reply:
Yea for sure but we need to refactor this to have 2-3 dependencies max.  There is way too many dependencies for a user to track, this doesn't fit well under the usability umbrella.  I only mention this because even I have gotten caught up by this and it's just a nuisance.